torchio/data/image.py

```diff
@@ -61,6 +61,11 @@ class Image(dict):
         tensor: If :py:attr:`path` is not given, :attr:`tensor` must be a 4D
             :py:class:`torch.Tensor` or NumPy array with dimensions
             :math:`(C, W, H, D)`.
+        h5DS: If both path and tensor aren't supplied, then an HDF5 Dataset has
+            to be supplied. 
+        lazypatch: Only applicable with h5DS. If ``True``, the h5DS won't be 
```

Comment:
Can you rename this to snake case? I.e., `lazy_patches`.

Reply:
Sure